src/form.js

```diff
@@ -0,0 +1,51 @@
+import React, {Component} from 'react'
+
+const styles = {
+  form: {
+    width: '25rem',
+    marginLeft: 'auto',
+    marginRight: 'auto',
+    border: '1px solid black',
```

Comment:
The default border in Bootstrap goes better with the rest of their styles.
https://getbootstrap.com/docs/4.1/utilities/borders/

If you want a more unique set of styles, you can choose Bootstrap a theme from https://bootswatch.com/ and link it from https://cdnjs.com/libraries/bootswatch